Description: find documentation of applications built against Debian KDE SC << 4.4.4
 Debian has switched HTML_INSTALL_DIR to /usr/share/doc/kde with the release of
 KDE SC 4.4.4 to Debian unstable. This patch allows khelpcenter4 to open
 documentation of KDE applications that were built with prior versions of KDE 4
 Develop Plarform.
Author: Modestas Vainius <modax@debian.org>
Author: Xavier Vello <xavier.vello@gmail.com>
Forwarded: not-needed
Origin: vendor
Last-Update: 2010-05-28

Index: kde4libs/kdecore/kernel/kstandarddirs.cpp
===================================================================
--- kde4libs.orig/kdecore/kernel/kstandarddirs.cpp	2010-11-25 16:40:08.000000000 +0100
+++ kde4libs/kdecore/kernel/kstandarddirs.cpp	2010-11-25 16:40:12.000000000 +0100
@@ -1856,6 +1856,11 @@
 
     addResourceType("autostart", "xdgconf-autostart", "/"); // merge them, start with xdg autostart
     addResourceType("autostart", NULL, "share/autostart"); // KDE ones are higher priority
+
+    // Debian patch: add old KDE4 HTML documentation dir, so that
+    // khelpcenter4 can open documentation of KDE applications built against
+    // Debian KDE SC << 4.4.4.
+    addResourceDir("html", "/usr/share/doc/kde4/HTML", false);
 }
 
 static QStringList lookupProfiles(const QString &mapFile)
